### ISSUE TITLE
dns.resolver.NoResolverConfiguration: cannot open /etc/resolv.conf {TERMUX}

### ISSUE THREAD

**Issue by essoojay** [guelfoweb/knock#149]
Error trying to run on termux:

phone rooted (if that helps)

dns.resolver.NoResolverConfiguration: cannot open /etc/resolv.conf

how to fix this 

thanks

**Comment by dfalt0** [comment]
If I remember correctly, Temux doesn't have the same file path for /etc/resolv.conf found in "traditional" linux - doing some googling, found some useful info relating to that: 

"Termux has /data/data/com.termux/files/usr/etc/resolv.conf - this file
is used by "dig" and other utilities which perform hostname resolution
internally. Others utilities use the DNS servers provided by Android OS."
https://groups.io/g/termux/topic/where_is_etc_resolve_conf/71116836?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,71116836 

Basically, knockpy isn't compatible with Temux out of the box
